share/man/man7/hier.7

```diff
@@ -581,9 +581,7 @@ system messages database;
 see
 .Xr msgs 1
 .It Pa preserve/
-temporary home of files preserved after an accidental death
-of an editor;
-see
+temporary storage for files following a crash of an editor; see
 .Xr vi 1
```

Comment:
[vi(1)](https://man.freebsd.org/cgi/man.cgi?query=vi&sektion=1&manpath=freebsd-release) makes no mention of _preserve_. 

Let's have a truer description for `/var/preserve/`.

Reply:
… how about: 

> reserved for historical reasons

----

For what it's worth, in a Linux context, <https://refspecs.linuxfoundation.org/FHS_2.3/fhs-2.3.html#VARLIBLTEDITORGTEDITORBACKUPFILESAN> includes this rationale under `/var/lib/<editor>`: 

> Previous Linux releases, as well as all commercial vendors, use `/var/preserve` for vi or its clones. However, each editor uses its own format for these crash-recovery files, so a separate directory is needed for each editor.
> 
> Editor-specific lock files are usually quite different from the device or resource lock files that are stored in `/var/lock` and, hence, are stored under `/var/lib`.

Reply:
Some small note about how it is unused but historical would be adequate IMO.

Probably we could go further and purge it from the src tree, including the (useless?) clean-preserve `periodic(8)` script. I don't think there would be any harm done by losing this historical information either.

Reply:
Thanks; 

* [`5ba7c58` (#763)](https://github.com/freebsd/freebsd-src/pull/763/commits/5ba7c585d11a38083de155d0495077c3e387bce6)
* FreeBSD bug [271933 – Consider removal of /usr/share/misc/fonts/, /var/preserve/ and the clean-preserve periodic(8) script](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=271933).